stylesheets/CSS/typography.css

```diff
@@ -48,4 +48,14 @@ p {
     font-weight: 400;
     line-height: 1.5rem;
     font-size: 1rem;
+}
+.btn{
+    font-family: var(--st-special-text);
+    font-size: 1.25rem;
+    font-weight: 600;
+}
+.link-text{
+    font-family: var(--st-body-text);
+    font-size: 1rem;
+    font-weight: 500;
 }
```

Comment:
If you are using multiple font families, you can directly use their names as that would make more sense.